pycbc/workflow/jobsetup.py

```diff
@@ -696,35 +700,27 @@ def create_node(self, data_seg, valid_seg, parent=None, inj_file=None,
             raise ValueError("The option pad-data is a required option of "
                              "%s. Please check the ini file." % self.name)
 
-        # FIXME: Should be done using file_input options in the config file
-        # Feed in bank_veto_bank.xml
-        if self.cp.has_option('inspiral', 'do-bank-veto'):
-            if not bankVetoBank:
-                raise ValueError("%s must be given a bank veto file if the "
-                                 "argument 'do-bank-veto' is given"
-                                 % self.name)
-            node.add_input_opt('--bank-veto-templates', bankVetoBank)
-
+        # Feed in bank_veto_bank.xml, if given
+        if self.cp.has_option('workflow-inspiral', 'bank-veto-bank-file'):
+            node.add_input_opt('--bank-veto-bank-file', bankVetoBank)
         # Set time options
         node.add_opt('--gps-start-time', data_seg[0] + int(pad_data))
         node.add_opt('--gps-end-time', data_seg[1] - int(pad_data))
         node.add_opt('--trig-start-time', valid_seg[0])
         node.add_opt('--trig-end-time', valid_seg[1])
+        node.add_opt('--trigger-time', self.cp.get('workflow', 'trigger-time'))
 
         # Set the input and output files
         node.new_output_file_opt(data_seg, '.hdf', '--output',
                                  tags=tags, store_file=self.retain_files)
         node.add_input_opt('--bank-file', parent, )
 
-        # TODO: isn't there a cleaner way of doing this?
-        # FIXME: YES! Don't use frame-cache!!!
         if dfParents is not None:
-            node.add_arg('--frame-cache %s' % \
-                         " ".join([":".join([frameCache.ifo, frameCache.name])\
-                                   for frameCache in dfParents]))
-            for frameCache in dfParents:
-                node.add_input(frameCache)
-            #node.add_input_list_opt('--frame-cache', dfParents)
+            frame_arg = '--frame-files'
+            for frame_file in dfParents:
+                frame_arg += ' ' + ':'.join([frame_file.ifo, frame_file.name])
```

Comment:
Much simpler with an f-string:
```suggestion
                frame_arg += f' {frame_file.ifo}:{frame_file.name}'
```